uint8 EXECUTE        = 0
uint8 USES_LEFT_ARM  = 1
uint8 USES_RIGHT_ARM = 2
uint8 USES_LEFT_LEG  = 3
uint8 USES_RIGHT_LEG = 4
uint8 USES_TORSO     = 5

float32[] commands
float32[] times
uint8 flags
uint8 slot
---
std_msgs/Empty empty